gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -359,7 +359,12 @@ it will appear in the Runtime Manager UI even if it is no longer running.
 [[helm-chart-options]]
 === Helm Chart Configuration Options
 
-The following command describes the configurable options of the Flex Gateway Ingress Controller Helm chart:
+The following command describes the configurable options of the Flex Gateway Helm chart:
```

Comment:
This reads funny because the syntax is not right the lead-in. Instead there are more words after it. It would be good if you could incorporate the text in the second sentence with the lead-in. Maybe you meant to say "The following commands" instead? If so, end it with a period.